XamarinCommunityToolkit/Views/CameraView/CameraView.shared.cs

```diff
@@ -0,0 +1,119 @@
+
+		public bool IsBusy
+		{
+			get => (bool)GetValue(IsBusyProperty);
+			set => SetValue(IsBusyProperty, value);
+		}
+
+		public static readonly BindableProperty IsAvailableProperty = BindableProperty.Create(nameof(IsAvailable), typeof(bool), typeof(CameraView), false,
+			propertyChanged: (b, o, n) => ((CameraView)b).OnAvailable?.Invoke(b, (bool)n));
+
+		public bool IsAvailable
+		{
+			get => (bool)GetValue(IsAvailableProperty);
+			set => SetValue(IsAvailableProperty, value);
+		}
+
+		public static readonly BindableProperty CameraOptionsProperty = BindableProperty.Create(nameof(CameraOptions), typeof(CameraOptions), typeof(CameraView), CameraOptions.Default);
+
+		public CameraOptions CameraOptions
+		{
+			get => (CameraOptions)GetValue(CameraOptionsProperty);
+			set => SetValue(CameraOptionsProperty, value);
+		}
+
+		public static readonly BindableProperty SavePhotoToFileProperty = BindableProperty.Create(nameof(SavePhotoToFile), typeof(bool), typeof(CameraView), false);
+
+		public bool SavePhotoToFile
+		{
+			get => (bool)GetValue(SavePhotoToFileProperty);
+			set => SetValue(SavePhotoToFileProperty, value);
+		}
+
+		public static readonly BindableProperty CaptureOptionsProperty = BindableProperty.Create(nameof(CaptureOptions), typeof(CameraCaptureOptions), typeof(CameraView), CameraCaptureOptions.Default);
+
+		public CameraCaptureOptions CaptureOptions
+		{
+			get => (CameraCaptureOptions)GetValue(CaptureOptionsProperty);
+			set => SetValue(CaptureOptionsProperty, value);
+		}
+
+		public static readonly BindableProperty KeepScreenOnProperty = BindableProperty.Create(nameof(KeepScreenOn), typeof(bool), typeof(CameraView), false);
+
+		public bool KeepScreenOn
+		{
+			get => (bool)GetValue(KeepScreenOnProperty);
+			set => SetValue(KeepScreenOnProperty, value);
+		}
+
+		public static readonly BindableProperty VideoStabilizationProperty = BindableProperty.Create(nameof(VideoStabilization), typeof(bool), typeof(CameraView), false);
+
+		public bool VideoStabilization
+		{
+			get => (bool)GetValue(VideoStabilizationProperty);
+			set => SetValue(VideoStabilizationProperty, value);
+		}
+
+		public static readonly BindableProperty FlashModeProperty = BindableProperty.Create(nameof(FlashMode), typeof(CameraFlashMode), typeof(CameraView), CameraFlashMode.Off);
+
+		public CameraFlashMode FlashMode
+		{
+			get => (CameraFlashMode)GetValue(FlashModeProperty);
+			set => SetValue(FlashModeProperty, value);
+		}
+
+		public static readonly BindableProperty PreviewAspectProperty = BindableProperty.Create(nameof(PreviewAspect), typeof(Aspect), typeof(CameraView), Aspect.AspectFit);
+
+		public Aspect PreviewAspect
+		{
+			get => (Aspect)GetValue(PreviewAspectProperty);
+			set => SetValue(PreviewAspectProperty, value);
+		}
+
+		public static readonly BindableProperty ZoomProperty = BindableProperty.Create(nameof(Zoom), typeof(float), typeof(CameraView), 1f);
+
+		public float Zoom
```

Comment:
Would you like to replace useless "float" by super-duper "double" ?)

Reply:
@AndreiMisiukevich @jfversluis UWP appears to use float vs double in it's camera control.  It's only the Zoom setting, so I can just cast back from double to float if you still think it best to keep double on CameraView.